tools/xskperfsuite.ps1

```diff
@@ -156,9 +159,15 @@ try {
     }
 
     Write-Verbose "installing xdp..."
-    & "$RootDir\tools\setup.ps1" -Install xdp -Config $Config -Arch $Arch
+    & "$RootDir\tools\setup.ps1" -Install xdp -XdpInstaller INF -Config $Config -Arch $Arch -EnableKmXdpApi:$KernelMode
```

Comment:
why install using INF?

Reply:
@nigriMSFT ?

Reply:
I don't think I had any valid reason for this. Was probably just hacking around some inconvenience I didn't spend time fixing at the time